### ISSUE TITLE
Language 'de' does not implement 'ShortIntegerContent'.

### ISSUE THREAD

**Issue by seekM** [dictation-toolbox/dragonfly#94]
After updating dragonfly I get the following error when starting Caster with a german DNS profile:

```
WARNING:timer:Dragonfly's _Timer class has been deprecated. Please use engine.create_timer() instead.
Ignoring ccr rule 'VHDL'. Failed to load with: 
<class 'dragonfly.error.DragonflyError'>
Language 'de' does not implement 'ShortIntegerContent'.
<traceback object at 0x0ED5F350>
  File "C:\dev\Natlink-User\Caster\castervoice\lib\ccr\__init__.py", line 36, in <module>
    [class_name])  # attempts to import the class

  File "C:\dev\Natlink-User\Caster\castervoice\lib\ccr\vhdl\vhdl.py", line 6, in <module>
    from castervoice.lib.imports import *

  File "C:\dev\Natlink-User\Caster\castervoice\lib\imports.py", line 10, in <module>
    from castervoice.lib import control, utilities, text_manipulation_functions

  File "C:\dev\Natlink-User\Caster\castervoice\lib\text_manipulation_functions.py", line 6, in <module>
    from castervoice.lib.ccr.core.punctuation import text_punc_dict,  double_text_punc_dict

  File "C:\dev\Natlink-User\Caster\castervoice\lib\ccr\core\punctuation.py", line 61, in <module>
    class Punctuation(MergeRule):

  File "C:\dev\Natlink-User\Caster\castervoice\lib\ccr\core\punctuation.py", line 85, in Punctuation
    IntegerRefST("npunc", 0, 10),

  File "C:\dev\Natlink-User\Caster\castervoice\lib\dfplus\additions.py", line 39, in __init__
    content = language.ShortIntegerContent

  File "C:\dev\Python27\lib\site-packages\dragonfly\language\loader.py", line 70, in __getattr__
    return self.get_attribute(name)

  File "C:\dev\Python27\lib\site-packages\dragonfly\language\loader.py", line 84, in get_attribute
    % (self._language, name))
```

**Comment by Danesprite** [comment]
Thanks for reporting this problem. I am unable to implement this because I don't know German, but I'm happy to take pull requests on it and other features that are unimplemented for specific languages.

In the meantime, a simple fix would be for `ShortIntegerContent` to alias `IntegerContent` for each unimplemented language. Perhaps it should also emit a warning message when it is first used/imported? It could also be fixed within Caster by catching the `DragonflyError`.

@mrob95 Does this fix sound reasonable to you?

**Comment by LexiconCode** [comment]
A possible workaround but you'll have to verify by testing.

Under `"miscellaneous"`in settings.toml set `"short_integer_opt_out": False,` to `True`

**Comment by Danesprite** [comment]
This was fixed by 5a62892. Dragonfly will fallback to `IntegerContent` if you try to use `ShortIntegerRef` with a speaker language that isn't English. A warning message will also be printed. This applies to all engines.

As I have no way of testing with a German DNS profile, I tested this by adding an optional `--language` argument to the `python -m dragonfly test` command:

```Shell
python -m dragonfly test -n --language en _test.py  # English (default)
python -m dragonfly test -n --language de _test.py  # German

```

The `_test.py` file just initialised a `ShortIntegerRef` object. As stated in the docs, the new argument only applies to the text-input engine.

Should be releasing version 0.16.0 this weekend. Will close this afterwards.
